library_generation/owlbot/bin/entrypoint.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+scripts_root=$1
+versions_file=$2
+synthtool_image_id=$3
+workspace=$(pwd)
+
+function runWithSynthtool() {
```

Comment:
This is one of the main differences with the original owlbot java - we run the owlbot java python scripts with a container that has synthtool in its environment